ios/Video/RCTVideo.swift

```diff
@@ -611,10 +627,86 @@ class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverH
                 playerItem.navigationMarkerGroups = RCTVideoTVUtils.makeNavigationMarkerGroups(chapters)
             }
         #endif
-
         return playerItem
     }
 
+    func getAudioTrackInfo(
+        model _: M3U8PlaylistModel,
+        principalModel: M3U8PlaylistModel
+    ) -> [String: Any] {
+        var streamList: NSArray = .init()
+
+        for i in 0 ..< principalModel.masterPlaylist.xStreamList.count {
+            let inf = principalModel.masterPlaylist.xStreamList.xStreamInf(at: i)
+            if let inf {
+                streamList.adding(inf)
+            }
+        }
+
+        if let currentEvent: AVPlayerItemAccessLogEvent = _player?.currentItem?.accessLog()?.events.last {
+            let predicate = NSPredicate(format: "%K == %f", "bandwidth", currentEvent.indicatedBitrate)
+            let filteredArray = streamList.filtered(using: predicate)
+            let current = filteredArray.last
+
+            if let current = current as? M3U8ExtXStreamInf {
+                let mediaList: NSArray = .init()
+                for i in 0 ..< principalModel.masterPlaylist.xMediaList.audio().count {
+                    let inf = principalModel.masterPlaylist.xMediaList.audio().xMedia(at: i)
+                    if let inf {
+                        mediaList.adding(inf)
+                    }
+                }
+
+                let predicate = NSPredicate(format: "SELF.groupId == %@", current.audio)
+                let currentAudio = mediaList.filtered(using: predicate).last
+
+                if let currentAudio = currentAudio as? M3U8ExtXMedia {
+                    let url: URL = currentAudio.m3u8URL()
+                    let audioModel: M3U8PlaylistModel? = try? .init(url: url)
+
+                    if let audioModel {
+                        let audioInfo: M3U8SegmentInfo = audioModel.mainMediaPl.segmentList.segmentInfo(at: 0)
+
+                        return [
```

Comment:
trzeba przejrzeć się temu co jest w dokumentacji https://miquido.github.io/react-native-video/component/events#onaudiotracks kontra temu co jest zwracane tutaj. Bo to nie do końca wygląda na spójne. 

zastanawiające jest też to że onAudioTracks było jakby zaimplementowane, a tutaj wygląda jakby osobne miejsce skąd to dostarczamy. Nie możemy jakość tych miejsc pogodzić?

Reply:
Nie widzę zmian w `docs/pages/component/events.mdx`.

Obecnie to wygląda tak https://miquido.github.io/react-native-video/component/events#onaudiotracks

A ma być chyba tak

Property | Type | Description | Change
-- | -- | -- | --
index | number | Track index | -
trackId | string | Track ID of the video track (android only) | - 
title | string | Descriptive name for the track | - 
language | string | 2 letter ISO 639-1 code(opens in a new tab) representing the language | -
bitrate | number | bitrate of track | dodać `(android only)`
type | string | Mime type of track | dodać `(android only)`
selected | boolean | true if track is playing | -
file | string | File name from DASH manifest (android only) | usunąć (android only)
supplementalProperties | string | Supplemental properties for the adaptation set from DASH manifest (android only) | -
codecs | string | Track codecs | nowy wiersz mogę dodać takie info na Android aby było na obu platformach |
channels | string | Tack channels (ios only) | nowy wiersz